devices/bitron.js

```diff
@@ -79,7 +79,7 @@ module.exports = [
         model: 'AV2010/29A',
         vendor: 'Bitron',
         description: 'SMaBiT Zigbee outdoor siren',
-        fromZigbee: [fz.ias_no_alarm],
+        fromZigbee: [fz.ias_siren],
```

Comment:
The existing converter should not be removed, otherwise you will get a `No converter available..` for the `ssIasZone`.`attributeReport`